src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
 spring.application.name=GitHubAiAPI
+
+
+Spring.demo.value= test
```

Comment:
It's a good practice to avoid unnecessary blank lines. Consider removing the blank lines at line 2 and 3.